.rubocop.yml

```diff
@@ -31,6 +31,9 @@ Layout/LineLength:
   Exclude:
     - db/**/*
 
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
```

Comment:
did this come from the update task?? I'm not sure we want this yet.. https://gist.github.com/fxn/bf4eed2505c76f4fca03ab48c43adc72#can-i-delete-the-magic-comments-in-ruby-34

Reply:
I was confused about this. Thought ruby 3.4 removed the need of having them. 
It seems you need to set `--enable-frozen-string-literal`  to use that option globally.
Thanks for the catch!